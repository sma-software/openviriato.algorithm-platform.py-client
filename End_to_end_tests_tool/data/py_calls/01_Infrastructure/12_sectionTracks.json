{
    "PythonParameterMapping": {
      "section_track_id" : "$.urlParameters.[0]"
    }
}
