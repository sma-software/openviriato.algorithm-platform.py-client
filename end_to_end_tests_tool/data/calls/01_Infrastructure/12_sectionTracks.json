{
    "restResource": "/section-tracks/{sectionTrackID}",
    "httpMethod": "GET",
    "urlParameters": [ "1082" ],
    "requestBody": null,
    "CSharpMethodName": "GetSectionTrack(sectionTrackID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory": "Get Section Track"
}
