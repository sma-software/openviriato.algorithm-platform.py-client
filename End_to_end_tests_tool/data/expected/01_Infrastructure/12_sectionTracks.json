{
  "id": 1082,
  "code": "900",
  "sectionCode": "70015",
  "weight": 10000,
  "debugString": "sectiontrack:s_70015 n_85ZHDB 900"
}